src/js/components/Button/Button.stories.tsx

```diff
@@ -66,97 +66,53 @@ stories.add("TeamSnap Colors", () => {
   );
 });
 
-stories.add("Status Colors", () => {
```

Comment:
Just looking in Storybook and we have two stories for colors: Themed colors and TeamSnap colors. I think we can consolidate this to just Colors since we will be making a theme for TeamSnap, and show all the buttons colors there.

Reply:
For that matter, I'm not sure we need the Default story either. That could be included in the colors story:
default, primary, positive, warning, negative